bond_app/fence_api.py

```diff
@@ -18,10 +18,16 @@ def get_credentials_google(self, access_token):
         """
         headers = {'Authorization': 'Bearer ' + access_token}
         result = requests.post(url=self.credentials_google_url, headers=headers)
+        # Logging the `result.content` here as part of debugging/troubleshooting for:
+        # https://broadworkbench.atlassian.net/browse/CA-1109
+        # Logging of the `response.content` can probably be removed after we have resolved this issue
+        logging.info("Getting new Service Account JSON Key from Fence via - request: POST {} - status code: {} - body: {}" \
+                     .format(self.credentials_google_url, result.status_code, result.content))
```

Comment:
The content being logged here is a service account key with access to protected data! Definitely don't log that.